soda/cmd/drop.go

```diff
@@ -12,20 +10,21 @@ var all bool
 var dropCmd = &cobra.Command{
 	Use:   "drop",
 	Short: "Drops databases for you",
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
The reason this was printing errors and not returning them was because in the case of `-a` it's common that a database like `production` can't be dropped or created, so returning the error could prevent the other databases from being created.